src/plugins/index_pattern_field_editor/public/components/field_editor/form_fields/script_field.tsx

```diff
@@ -91,7 +91,8 @@ export const ScriptField = React.memo(({ existingConcreteFields, links, syntaxEr
                 const painlessSyntaxErrors = PainlessLang.getSyntaxErrors();
                 // It is possible for there to be more than one editor in a view,
                 // so we need to get the syntax errors based on the editor (aka model) ID
-                const editorHasSyntaxErrors = editorId && painlessSyntaxErrors[editorId].length > 0;
+                const editorHasSyntaxErrors =
+                  editorId && painlessSyntaxErrors[editorId]?.length > 0;
```

Comment:
@sebelga I found this change was necessary, do you know why?

Reply:
That's strange. Let's bring @alisonelizabeth, maybe she knows how it is possible to not have the `editorId` key in the `painlessSyntaxErrors` map.

Reply:
That is strange. Is this only surfacing in the context of the functional test? I can try to run through the test tomorrow (ran out of time today) and see if I can figure out why this might be happening.

Reply:
I believe `painlessSyntaxErrors[editorId]` was evaluating to `undefined`

Reply:
removing the change seems to be a reliable way of creating the problem.....only in CI. Although I did see it once or twice running CI tests locally.

Reply:
Not sure it deserves too much time to spend on this as the fix from Matt does not hurt. It just seemed strange that the `painlessSyntaxErrors` map was not automatically assigned an editorId key with an empty array whenever an editor is created. Maybe there is more and it is added and then removed?

Reply:
You're correct @sebelga. I looked at the code again and the model id was not getting assigned to the errors object when it is created. I opened https://github.com/mattkime/kibana/pull/12 to address this. @mattkime if you're comfortable with the changes, feel free to merge into your branch. Or I can open up a separate PR against master if you prefer.

Reply:
Great, thanks for the fix @alisonelizabeth !